intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigDropDown.kt

```diff
@@ -57,9 +61,21 @@ class MirrordConfigDropDown : ComboBoxAction() {
                 return
             }
 
-            initializeConfigFiles(project)
+            if (!::configFiles.isInitialized) {
+                val basePath = project.basePath ?: throw Error("couldn't resolve project path")
+                configFiles = FileBasedIndex.getInstance().getAllKeys(MirrordConfigIndex.key, project).filter {
+                    matches(it, basePath)
```

Comment:
Do the files have to be filtered here again after being filtered in `MirrordConfigIndex` because now we have the base path?

Reply:
fixed